### PR TITLE
Knex seeds update

### PR DESCRIPTION
**Pull request by paulabernathy7**
# Description
Updated migration tables and seed data

Fixes # (issue)

## Type of change
Nonbreaking 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change Status

- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [ ] Test A
Screen Shot 2019-11-23 at 7 16 21 PM
Screen Shot 2019-11-23 at 7 16 35 PM

- [ ] Test B

# Checklist

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My code has been reviewed by at least one peer
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts

### CODE REVIEW

File: data/seeds/004-foods.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          polyunsaturated_fat_g: 0.339,
+          protein_g: 1.91,
+          carbs_g: 12.65,
+          sugar_g: 1.08,
+          fiber_g: 0.6,
+          sodium_mg: 170,
+          vitamin_a_daily_pct: 0,
+          vitamin_c_daily_pct: 0,
+          iron_daily_pct: 5,
+          calcium_daily_pct: 0
```
Comment (dave-vazquez) [inline data/seeds/004-foods.js]: Feed me seeeeeds 🎉

File: data/migrations/20191029215637_foods.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ exports.up = function(knex) {
     tbl.datetime("retrieved_at").notNullable();
     tbl.decimal("calories_kcal");
     tbl.decimal("fat_g");
+    tbl.decimal("trans_fat_g");
```
Comment (dave-vazquez) [inline data/migrations/20191029215637_foods.js]: I think you got trans_fat_g in here twice.

Comment (willediger) [reply data/migrations/20191029215637_foods.js]: @dave-vazquez @joeluna17 @paulabernathy7 

just pushed a commit that removes the duplicate trans_fat_g field.

**Comment by willediger** [comment]
> Awesome job!! Great catch on the "recipes" migration.
> 
> I was able to run this series of commands successfully on my local pg install, no problem:
> 
> `knex migrate:rollback; knex migrate:latest; knex migrate:rollback; knex migrate:latest; knex seed:run`

Note: I have very little trust of Knex not to have weird errors pop up. Thus, doing 2x rollback/latest migrations, to make sure everything is working well. In case anyone was wondering.

**Pull request merged**